src/frontend/app.ts

```diff
@@ -65,8 +66,9 @@ export default (octokitApp: App): Express => {
   const app = express();
   const rootPath = path.resolve(__dirname, '..', '..');
 
+  initializeSentry();
   // The request handler must be the first middleware on the app
-  app.use(Sentry.Handlers.requestHandler());
+  app.use(Sentry.Handlers.requestHandler() as express.RequestHandler);
```

Comment:
You shouldn't need this if you already have `@types/sentry` included in your project because the type is the same as `express.RequestHandler`.  Can you check to make sure your vscode isn't ignoring the `@types` directory because by [default they should be included](https://www.typescriptlang.org/tsconfig#types).

Reply:
This would also explain the issue you had with `req.session[state]`.